airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -105,6 +106,7 @@ def execute(self, context):
             self.jobId = response['jobId']
             self.jobName = response['jobName']
 
+            sleep(randint(10, 60))
```

Comment:
Does it really need to be random? - also one `self.log` before this would be nice imo

Reply:
- There is already a log, i.e. `self.log.info('AWS Batch Job started: %s', response)`
- The use of a random interval decreases the chances of exceeding an AWS API throttle limit
  - concern about it is reasonable, it's there as an extra measure to avoid throttle limits when there are 100's or 1000's of concurrent tasks all trying to start up batch jobs at about the same time, so a relatively wide window was selected in the hope that it can stagger the API calls randomly within about a minute, assuming the API throttle limits are assessed per second; the initial delay of at least 10 sec allows the batch job a bit of time to spin up, maybe it could be longer, but there it is.
- this call to the random-sleep should move to within the wait-method (in a follow-up amended commit)

Reply:
amended commit pushed

Reply:
Thank you very much for the explanation. LGTM then 👍 :)